src/StorageSystemsSimulations.jl

```diff
@@ -53,6 +54,7 @@ import DataStructures: OrderedDict
 const MOI = MathOptInterface
 const PSI = PowerSimulations
 const PSY = PowerSystems
+const PNM = PowerNetworkMatrices
```

Comment:
Lets do this to avoid dependency hell 

```suggestion
const PNM = PSI.PNM
```